packages/shared/src/hooks/useContextMenu.ts

```diff
@@ -13,20 +15,38 @@ interface UseContextMenu {
 export default function useContextMenu({
   id,
 }: UseContextMenuProps): UseContextMenu {
+  const key = useMemo(
+    () => generateQueryKey(RequestKey.ContextMenu, null, id),
```

Comment:
Probably tag user here

Reply:
Since we don't actually set any user related state, its just boolean.. I would say its ok for this not to be user tagged.